src/test/java/com/hedera/hashgraph/sdk/IdUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package com.hedera.hashgraph.sdk;
+
+import com.hedera.hashgraph.sdk.account.AccountId;
+import com.hedera.hashgraph.sdk.contract.ContractId;
+import com.hedera.hashgraph.sdk.file.FileId;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class IdUtilTest {
+    @Test
+    @DisplayName("parses id string correctly")
+    void testParseString() {
+        long[] correctArray = {0,0,0};
+        long[] returnedArray = IdUtil.parseIdString("0.0.0");
+        assertEquals( correctArray[0], returnedArray[0]);
+        assertEquals( correctArray[1], returnedArray[1]);
+        assertEquals( correctArray[2], returnedArray[2]);
+    }
```

Comment:
'd test the failure paths as well but I wouldn't block this PR on that.

Reply:
Thanks for the suggestions! I've implemented all your advice except for the failure paths. I tried to use assertThrows but I'm not very familiar with Java tests. Are there any good examples you suggest looking at?

Reply:
Sure, there's an example here: https://github.com/hashgraph/hedera-sdk-java/blob/master/src/test/java/com/hedera/hashgraph/sdk/CallParamsTest.java#L163-L185

You simply pass the class of the exception you're expecting and then a lambda which should throw that exception; the exception itself is returned from the `assertThrows()` call which you can then assert the message (the extra `message` parameter on `assertThrows()` is the error message for if _that_ assertion fails, not the expected value of the assertion message; I made that mistake before).

Reply:
Thanks! I just added those failure tests.